Overlay add popups at action blitz

nathanstilwell opened the pull request:
This is the work that we did at the [Universal Health Care Action Blitz](https://www.eventbrite.com/e/universal-health-care-action-blitz-tickets-34680840327). I branched off of `origin/overlay` branch to leverage the work done there.

MrSnickers committed:
Adds index and logic for Google spreadsheet backend

rcscastillo committed:
Create status.json

rapicastillo committed:
Merge pull request #1 from rcscastillo/patch-1

Create status.json

rcscastillo committed:
Update status.json

rapicastillo committed:
Merge pull request #2 from rcscastillo/patch-2

Update status.json

nathanstilwell committed:
Moving Scripts to the bottom

I guess for nostalgia, since we don't really have much of a DOM to
render with JS. I'm also adding lodash to use _.template for the popups

nathanstilwell committed:
Changing the map style a little bit

- Toning down that stroke weight a little
- Adding a bunch of stuff for the Pop Ups
- Adding opacity for districts and I can add a little transition in
there

nathanstilwell committed:
I kind of moved everything

- Wrapped the code in an IIFE
- Declared vars at the top to be explicit about "Hoisting"
- Removed the inline geometry geo data for readability
- Fetching Senators from a JSON file instead of a Google Spreadsheet
- Broke up all the event handling into smaller refactored functions

rawild commented on map.js:
```diff
@@ -0,0 +1,135 @@
+    accessToken: 'pk.eyJ1IjoibXJzbmlja2VycyIsImEiOiJjajNjM2ozNmswMDB4MnBzMWh6ZzZnNXQ1In0.HK0L0_LWCcc8Qihk0gNpYg'
+  }).addTo(mymap);
+
+  var mapFetch = $.getJSON('/data/nys-senatemap.json');
```
@nathanstilwell i think I said trailing on slack - i meant leading.. the leading '/' in this fetch and the next line are what were causing me problems

nathanstilwell replied:
I'm interested to find out why this is causing you a problem. How are you running the code? For development I was running a python simple server (`python -m SimpleHTTPServer`). Are you opening the page in a browser (serving from the file system)?

nathanstilwell committed:
Less code is better code

- Cut pointless reference to tileLayer
- Took out needless mouseover / mouseout handlers
- Pointlessly changed the order of global vars
